.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "fpla-docker"]
 	path = fpla-docker
-	url = https://github.com/hmcts/fpla-docker
+	url = ../fpla-docker.git
```

Comment:
I'd be happy to use "../../hmcts/fpla-docker.git" (for clarity) instead, if the team prefers.

Reply:
I think it doesn't hurt to have a bit of clarity with this, so do submodule urls work the same way as file paths?

Reply:
if build is green then we are ok, jenkins pipeline rely on submodule and scripts there

Reply:
Cool. I change it.